recipe/meta.yaml

```diff
@@ -476,6 +476,9 @@ outputs:
         {% set skips = skips ~ " or test_BCELoss_weights_no_reduce_cuda" %}             # [unix and cuda_compiler_version != "None"]
         {% set skips = skips ~ " or test_ctc_loss_cudnn_tensor_cuda " %}                # [unix and cuda_compiler_version != "None"]
         {% set skips = skips ~ " or (TestTorch and test_index_add_correctness)" %}      # [unix and cuda_compiler_version != "None"]
+        # These tests require higher-resource or more recent GPUs than the CI provides
+        {% set skips = skips ~ " or (TritonCodeGenTests and test_sdpa_inference_mode_aot_compile)" %}   # [unix and cuda_compiler_version != "None"]
+        {% set skips = skips ~ " or (TestNN and test_grid_sample)" %}                                   # [unix and cuda_compiler_version != "None"]
```

Comment:
OK, I was just redownloading the logs to double-check why they failed. But was going to do the same.

Reply:
we can also just skip those other tests and add a note if the backport doesn't work

Reply:
The backport was easy, but I was double-checking the remaining failures before pushing.

Reply:
or patch to rename the lib each time
```
def test_mutable_custom_op_fixed_layout(self):
    unique_lib_name = f"mylib_{id(self)}"  # Make unique name using test instance id
    with torch.library._scoped_library(unique_lib_name, "DEF") as lib:
```
Although I'm surprised I can't find anything about these tests failing in issues upstream.